### ISSUE TITLE
Not a `document` API

### ISSUE THREAD

**Issue by martinthomson** [patcg-individual-drafts/topics#71]
If the information the API exposes is a property of the browser (or indeed the user), why use `document` rather than `navigator`?

**Comment by martinthomson** [comment]
Yes, fun.  The information source not being bound to the document is something that carries more weight (at least in my view).

As for conditional on context, calls to `navigator.geolocation.getCurrentPosition()` or `navigator.mediaDevices.getUserMedia()` produce results that depend on the calling context in some way, just to name two things that come to mind.

**Comment by jkarlin** [comment]
It's a good question. I brought it up to the TAG as part of the review https://github.com/w3ctag/design-reviews/issues/726.

**Comment by bershanskiy** [comment]
Please consider this: using `navigator.browsingTopics()` will enable exposing Topics API to workers in the future (e.g., as `WorkerNavigator.browsingTopics`). Making Topics API usable in background contexts makes sense because a page or a frame might legitimately want to offload the main context by putting all the ad-selection logic into a background worker. Yes, `document.browsingTopics()` is asynchronous (and I'm really glad about it), but the site-provided ad-selection logic might be time-consuming and might be extracted into a worker. Message passing between main context and worker might not be very elegant (but admittedly not too bad).

**Comment by martinthomson** [comment]
@domenic, I'm not sure that it is a superstition.  It's about asking the appropriate entity.  If you see `window` as being a representation of the browser, then I guess `window` is the right thing.  I had inferred (perhaps incorrectly) that `navigator` was the browser and `window` was the browsers projection of the current tab.

**Comment by domenic** [comment]
Mmm, they're both global objects to the current tab, so just using the actual global object instead of nesting under navigator makes the most sense.

**Comment by jkarlin** [comment]
So how does `window.browsingTopics` sound to folks? `window.topics` might be a bit risky in terms of collisions. @slightlyoff for input as well.

**Comment by RByers** [comment]
+1 for avoiding `window.topics`. I don't want a repeat of `window.Report` and given only a small number of developers are likely to ever write code calling this API, I don't think it's worth worrying about brevity.

**Comment by jkarlin** [comment]
In terms of precedent, I see cookies are on `document` (I think this is actually originally why floc, and now topics are there as they're also cross-site information based on origin) and so are [private state tokens](https://wicg.github.io/trust-token-api/#idl-index). 

Is there a document written somewhere with guidance with what should hang off of what object? Wiithout such a reference I'm unclear on how we should proceed here and would prefer to just keep it on `document`.
